### ISSUE TITLE
Failed to initialise Oculus! ovr_initialuze: -3005

### ISSUE THREAD

**Issue by JaseyMitch** [Nibre/MotherVR#110]
Hi. Trying to use this for the first time today after setting up my first Oculus Rift today, and I'm getting the popup message above. It also says:

Make sure the Oculus runtime is insalled correctly, and try again.

If I try with -steamvr in the launch options, it says "Hmd not found (108)".

Any help is much appreciated. This was one of the main reasons I got a Rift and it doesn't work. The Rift installed fine and I played through the demo fine.

**Comment by JaseyMitch** [comment]
As an additional note, if Steam VR is running (with the headset on I can see the room, use my controllers and launch applications etc.) and I use -steamvr in launch options for A:I, I still get an error about my Hmd not being found.

**Comment by JaseyMitch** [comment]
I seem to have fixed it. Others may find how useful:

It seems that you need to run a VR game bought on Steam first before attempting to use this mod. If you - like me - try to play A:I as your first Steam game once you've installed your rift, you may get error messages about Oculus runtime like I did.

While I was searching for a solution, I found a post from someone who had a similar issue to me, right after installing their first Rift, but it mysteriously seemed to fix itself after they had played DCS for a bit (a free VR game on Steam).

Personally, I played Subnautica for a while, after installing and running SteamVR (a restart after installing SteamVR is essential, even though it doesn't ask, or it didn't detect my Rift. Some other posts I found seem to confirm this).

After I closed Subnautica, I ran A:I, straight into VR without issue (with SteamVR running and detecting my headset), and I was giggling like an idiot in no time!

TLDR: For new Rift users, play a steam VR game first before trying to run A:I with this mod or it won't work.

Feel free to delete this whole post if you think it won't be useful to anyone, but it might be helpful for the A:I enthusiast who tries to boot up this mod as their first port of call after plugging their new Oculus in. I only had a quick look around the Nostromo, but I am amazed. Thank you for your hard work.

**Comment by Strawb77** [comment]
props for that- too many folks find their solution and then skip without telling what it was..

**Comment by Strawb77** [comment]
i`m trying to make this work on my vive, kinda getting there, but as you say, lots of trial and error along the way. 
i too am v grateful to Nibre for making this possible at all, imma send him/her some beer tokens :)

**Comment by JaseyMitch** [comment]
As a final note to all this - and after some reinstalling and mucking about with it all today - I think perhaps the issue was (very silly of me) that I installed SteamVR before installing my headset for the first time. When you do it that way round, SteamVR doesn't necessarily see the Rift headset, controllers or sensors. I noticed that if you install your Rift first, Steam will actually say it's noticed you have a VR headset and asks if you'd like to install SteamVR. A:I seemed to work fine after that with no need for further steps. So, for new Rift users who want to play A:I with this mod, install your Rift, install SteamVR, and lastly run A:I with the mod installed using '-steamvr' in launch options for the game. I haven't tried to launch it through Oculus software, and I'm not going to! I just want to play the game after all my faffing around making daft mistakes. Thanks Nibre - a PayPal tip was just sent.
